App/backend/tools/schema_multi_extractor.py

```diff
@@ -100,14 +100,22 @@ def extract_second_index(data_dict_name):
             fields_number = len(column_names)
```

Comment:
```suggestion
            fields_number = len(table['columns'])
```

Rimuovere anche la riga sopra a questa istruzione, ossia column_names = [column['name'] for column in table['columns']]